### ISSUE TITLE
Update web & studio dependencies

### ISSUE THREAD

**Issue by SvSven** [equinor/energyvision#1027]
We have had a "update freeze" as we got close to going live. We are now live and everything seems to work, so we should aim to get things up to date. 

Relates to #852 - which fixes some issues (like the "Delete all (incl. translations" action)

**We need to thoroughly test things before we push any updates to production**

**Comment by SvSven** [comment]
Upgrading `next-sanity` to v0.5 is blocked by #941

**Comment by SvSven** [comment]
Upgrading React to v18 has its own issue since it's a major update that touches every aspect of our code: #1041
